src/seedu/addressbook/data/person/Address.java

```diff
@@ -26,6 +32,15 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
         this.value = address;
+        //Preparation and conversion from string to array of string 
+        //Assignment of array element to Class elements
+        address = address.replaceAll(", ", " ");
```

Comment:
- This will introduce a bug. Shouldn't split from spaces. 

- I guess you did not run the regression. If not this bug would have been discovered. Please remember to run the regression when you do changes.